test/multinomial.jl

```diff
@@ -98,3 +98,17 @@ r = fit_mle(Multinomial, x, fill(2.0, size(x,2)))
 @test r.n == nt
 @test length(r) == length(p)
 @test_approx_eq_eps probs(r) p 0.02
+
+# behavior for n = 0
+d0 = Multinomial(0,p)
```

Comment:
Can you insert spaces after a comma to match the style of the rest of this file?